cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -569,15 +583,37 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
                 .build();
     }
 
+    private static String converttoformat(Map<String, Object> map) {
+
+        StringBuilder mapAsString = new StringBuilder("{");
+        for (String key : map.keySet()) {
+            mapAsString.append(key).append("\n").append(Objecttoarray(map.get(key))).append(",\n");
+        }
+        mapAsString.delete(mapAsString.length() - 2, mapAsString.length()).append("}");
+
+        return mapAsString.toString();
+    }
+
+    private static String Objecttoarray(Object obj) {
+
+        String myarray = "";
+
+        if (obj.getClass().isArray()) {
+            if (obj.getClass().getComponentType().equals(int.class)) {
+                int[] numbers = (int[]) obj;
+                myarray += "Before: " + numbers[0] + "\n" + "After: " + numbers[1] + "\n" + "Diff: " + numbers[2] + "\n";
+            } else {
+                double[] numbers = (double[]) obj;
+                myarray += "Before: " + numbers[0] + "\n" + "After: " + numbers[1] + "\n" + "Diff: " + numbers[2] + "\n";
+            }
+        }
+        return myarray;
+    }
+
     /**
-<<<<<<< HEAD
```

Comment:
You appear to have the remenants of a merge conflict in this PR?

Reply:
Yeah. These were already present when I started working on the PR